frontend/plugins/frontline_ui/src/modules/integrations/erxes-messenger/components/EMGreeting.tsx

```
@@ -0,0 +1,166 @@
+                      }));
+                    }}
+                  />
+                  <Form.Message />
+                </Form.Item>
+              )}
+            />
+            <Form.Item>
+              <Form.Label>Social Links</Form.Label>
+              {fields.map((field, index) => {
+                return (
+                  <div className="flex gap-2">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)